impl/ocaml/sqlgg_traits.ml

```diff
@@ -101,9 +102,9 @@ module type M = sig
 
   (** Execute non-query.
     @raise Oops on error
-    @return number of affected rows
+    @return execute_response
```

Comment:
either explain what is execute_response or remove the line (it is clear from the type that it returns `execute_response`)

Reply:
removed